### ISSUE TITLE
Lucene - fails to create folder automatically

### ISSUE THREAD

**Issue by MikeKry** [OrchardCMS/OrchardCore#9147]
When you setup lucene index and access project from different computer/folder, without sharing contents of folder "App_Data\Sites\Default\Lucene" then OC fails on error when I try to rebuild index. 

E.g. if you have index "Fulltext" and OC fails to build index, it is enough to manually create folder "Fulltext" in "App_Data\Sites\Default\Lucene", but I think, it should be checked and created automatically. 

In other words, there is missing check for existence of directory for Lucene Indices

**Comment by Skrypt** [comment]
duplicate.
